wordpress-mobile/gutenberg-mobile
# Add Reusable blocks to the inserter menu

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/28495

Closes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2575

Add support to insert existing Reusable blocks from the editor in the mobile version. For this purpose the inserter menu now shows two tabs in case this kind of blocks have been previously created in the site.

The Reusable tab will show a list of all created Reusable blocks of the site.

To test:
Check testing instructions in [`gutenberg` PR](https://github.com/WordPress/gutenberg/pull/28495).

PR submission checklist:
